lang/mlton/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+PortGroup           openssl 1.0
+
+github.setup        MLton mlton 63666ac8ab65da81b9ec9595e7053d26e0682f51
+version             20230721
+revision            0
+categories          lang ml
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+license             HPND
+description         Whole-program optimizing compiler for the SML
+long_description    MLton is a whole-program optimizing compiler for the Standard ML programming language. \
+                    MLton generates small executables with excellent runtime performance, utilizing untagged and unboxed native integers, \
+                    reals, and words, unboxed native arrays, fast arbitrary-precision arithmetic based on GMP, and multiple code generation \
+                    and garbage collection strategies. In addition, MLton provides a feature rich Standard ML programming environment, \
+                    with full support for SML97 as given in The Definition of Standard ML (Revised), a number of useful language extensions, \
+                    a complete implementation of the Standard ML Basis Library, various useful libraries, a simple and fast C foreign function interface, \
+                    the ML Basis system for programming with source libraries, and tools such as a lexer generator, a parser generator and a profiler.
+homepage            http://www.mlton.org
```

Comment:
Please prefer HTTPS

Reply:
It has no https. Try it, browser will not open the page, saying connection is not secure.

Reply:
![mlton](https://github.com/macports/macports-ports/assets/92015510/2b6746e8-6800-4978-addf-e86acb46bc01)